Sprint 16 bug fixes

lukybil opened the pull request:
- #188 disable sticky headers
- #271 render user location after other shapes
- #324 delete assistants from the whole app
- #313 The positioning in the middle was not easy because of the left arrow, the title was not centered, this could be solved with an absolutely posisitoned container, but this solution works without absolutely positoned elements
- #337 changed one `router.push` to `router.replace`

PavelFilo commented on app/(app)/parking-cards/verification/index.tsx:
```diff
@@ -31,7 +31,7 @@ const Page = () => {
       const tmpVerificationToken = res.data[0].token
 
-      router.push({
+      router.replace({
```
cannot be done like this, because if user wants to go back from verification-sent to email it wont be possible, just leave it with `push` and I will adjust new part of code I added to verification-result page :)

PavelFilo replied:
done here: https://github.com/bratislava/paas-mpa/pull/340

PavelFilo commented on components/screen-layout/StackScreenWithHeader.tsx:
```diff
@@ -1,27 +1,83 @@
+                marginRight: headerSideWidth - headerRightWidth,
+              }}
+            >
+              <Typography variant="h1" className="bottom-[3px] max-w-full text-ellipsis">
```
font size of header title is too big, compare it with design

PavelFilo reviewed (changes requested):
Make sure the header is done according to the design :)

PavelFilo commented:
Please test all the headers in the app if they have the correct title after these changes (I found one that hasn't, fixed it, and pushed it to this PR)

lukybil commented:
> Please test all the headers in the app if they have the correct title after these changes (I found one that hasn't, fixed it, and pushed it to this PR)

What do you mean by this please? I changed back the title that you changed. It should be passed to the `ScreenView` to render the custom title.

lukybil commented on app/(app)/(purchase-and-payment)/purchase/choose-vehicle.tsx:
```diff
@@ -53,10 +53,9 @@ const ChooseVehicleScreen = () => {
 
   return (
-    <ScreenView>
+    <ScreenView title={t('title')}>
```
Of course I have tested it. Aren't there also two `Stack.Screen` for all those screens where we set other attributes like animation or whether it's a modal in layout routes?
